mandatory/2-function-creation.js

```diff
@@ -16,17 +25,28 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
-
+function formatPercentage(num) {
+let newNum=Math.round(num*100)/100;
+if (newNum > 100) newNum=100;
```

Comment:
I would right this line of could before Math.round one, so if the number is bigger than 100 it wont go trough the whole process and just return 100%, I think it might lead the process time to be shorter and executed quicker.

Reply:
This is a good thing to be thinking about. How to "return early" from a function to avoid doing unecessary work.